app/assets/stylesheets/common/base/reviewables.scss

```diff
@@ -0,0 +1,221 @@
+.reviewable-title {
+  margin-bottom: 1.5em;
+}
+
+.reviewable-topics {
+  width: 100%;
+
+  tbody {
+    td {
+      padding: 0.5em;
+    }
+  }
+}
+
+.reviewable-filters {
+  background-color: $primary-very-low;
+  padding: 0.5em 1em;
+  margin-bottom: 1em;
+  display: flex;
+
+  .all-filters {
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+  }
+
+  .reviewable-filter {
+    display: flex;
+    align-items: center;
+    margin-right: 1em;
+
+    label {
+      margin: 0 0.5em 0 0;
+    }
+    input {
+      margin: 0;
+    }
+  }
+
+  .topic-filter {
+    margin-top: 1em;
+    .btn {
+      margin-left: 0.5em;
+    }
+  }
+  .refresh {
+    height: 1em;
+    display: flex;
+  }
+
+  .score-filter {
+    width: 5em;
+  }
+}
+.user-flag-percentage {
+  display: flex;
+  align-items: center;
+  margin-left: 0.5em;
+
+  .percentage-label {
+    margin-right: 0.25em;
+    &.agreed {
+      color: $success;
+    }
+    &.disagreed {
+      color: $danger;
+    }
+    &.ignored {
+      color: $primary-medium;
+    }
+  }
+  .d-icon {
+    font-size: 0.9em;
+  }
+}
+
+.reviewable-item {
+  margin-bottom: 1em;
+  border-bottom: 1px solid dark-light-choose($primary-low, $secondary-low);
+
+  .topic-statuses {
+    float: none;
+  }
+
+  .reviewable-meta-data {
+    color: dark-light-choose($primary-medium, $secondary-medium);
+    display: flex;
+    width: 100%;
+    margin-bottom: 0.5em;
+    font-size: $font-down-1;
+    align-items: center;
+    .reviewable-type {
+      margin-right: 1em;
+    }
+    .created-at {
+      a {
+        color: dark-light-choose($primary-medium, $secondary-medium);
+      }
+      margin-right: 1em;
+    }
+    .score {
+      margin-right: 1em;
+    }
+    .status {
+      color: dark-light-choose($primary-high, $secondary-high);
+    }
+  }
+
+  .reviewable-contents {
+    display: flex;
+  }
+
+  .reviewable-actions {
+    margin-top: 0.5em;
+    display: flex;
+
+    .reviewable-action,
+    .reviewable-action-dropdown {
+      margin-right: 0.5em;
+
+      &.delete-user {
+        @extend .btn-danger;
+      }
+    }
+  }
+  padding-bottom: 1em;
+}
+
+.reviewable-scores {
+  min-width: 50%;
+
+  .user {
+    display: flex;
+    align-items: center;
+
+    .user-flag-percentage {
+      margin-left: 0.5em;
+    }
+  }
+  th {
```

Comment:
same remark on html styles

Reply:
In this case I think it's more defensible, because a table has a lot of elements th, td that would all need a new class attribute. Instead I made it more explicit by introducing `> th > tr` so that the styles only apply to the format of this table. If there's a nested table they should not apply.

Reply:
makes sense :+1: